ammico/colors.py

```diff
@@ -91,7 +91,12 @@ def analyse_image(self):
                 merge_color=self.merge_color,
                 delta_e_method=self.delta_e_method,
             )
-            self.subdict[rgb_name] += round(color.proportion, 2)
+            self.subdict[rgb_name] += color.proportion
+
+        # round all colors
+        for key in self.subdict.keys():
+            if key != "filename":
+                self.subdict[key] = round(float(self.subdict[key]), 2)
```

Comment:
this is potentially dangerous, as there may already be other keys in the dictionary like `"faces"`, `"text_english"`, etc.

Better to change to something like
```
self.subdict= {key: round(float(self.subdict[key]), 2) if key in colors else self.subdict[key] for key in self.subdict.keys()}
```